app/tests/api/inventory.test.ts

```diff
@@ -110,6 +175,43 @@ describe("Inventory API", () => {
     assert.equal(data.totalEmissions, inventory.totalEmissions);
   });
 
+  it("should find an inventory with csv format", async () => {
+    const url = `http://localhost:3000/api/v0/city/${locode}/inventory/${inventory.year}?format=csv`;
+    const req = createRequest(url);
+    const res = await findInventory(req, {
+      params: { city: locode, year: inventory.year.toString() }
+    });
+    assert.equal(res.status, 200);
+    assert.equal(res.headers.get("content-type"), "text/csv");
+    assert.ok(res.headers.get("content-disposition")?.startsWith("attachment"));
+    const csv = await res.text();
+    const lines = csv.split("\n");
+    assert.ok(lines.length > 0);
+    const headers = lines[0].split(",");
+    assert.equal(headers.length, 6);
+    assert.equal(headers[0], "Inventory Reference");
+    assert.equal(headers[1], "Total Emissions");
+    assert.equal(headers[2], "Activity Units");
+    assert.equal(headers[3], "Activity Value");
+    assert.equal(headers[4], "Emission Factor Value");
+    assert.equal(headers[5], "Datasource ID");
+    assert.ok(lines.length > 1);
+    assert.strictEqual(lines.length, 3);
+    assert.ok(lines.slice(1).every(line => line.split(",").length == 6));
+  });
+
+  it("should find an inventory with xls format", async () => {
+    const url = `http://localhost:3000/api/v0/city/${locode}/inventory/${inventory.year}?format=xls`;
```

Comment:
Since this URL actually doesn't serve a function in the tests, I've resulted to make a new function `mockRequest` that doesn't take a URL as the argument. Otherwise this would be hard to keep in sync since it's not checked/ enforced by the test or route for any test case.